modules/monitoring-investigating-why-user-defined-metrics-are-unavailable.adoc

```diff
@@ -17,14 +17,42 @@ endif::openshift-rosa,openshift-rosa-hcp,openshift-dedicated[]
 ifdef::openshift-rosa,openshift-rosa-hcp,openshift-dedicated[]
 * You have access to the cluster as a user with the `dedicated-admin` role.
 endif::openshift-rosa,openshift-rosa-hcp,openshift-dedicated[]
-* You have installed the OpenShift CLI (`oc`).
+* You have installed the {oc-first}.
 * You have enabled and configured monitoring for user-defined projects.
 * You have created a `ServiceMonitor` resource.
 
 .Procedure
 
-. *Check that the corresponding labels match* in the service and `ServiceMonitor` resource configurations.
-.. Obtain the label defined in the service. The following example queries the `prometheus-example-app` service in the `ns1` project:
+. Ensure that your project is not excluded from user workload monitoring. The following examples use the `ns1` project.
+
+.. Verify that the project _does not_ have the `openshift.io/user-monitoring=false` label attached:
++
+[source,terminal]
+----
+$ oc get namespace ns1 --show-labels | grep 'openshift.io/user-monitoring=false'
+----
++
+[NOTE]
+====
+The default label set for user workload projects is `openshift.io/user-monitoring=true`. However, the label is not visible unless you manually apply it.
+====
+
+.. If the label is attached, remove the label:
++
+.Example of removing the label from the project
```

Comment:
```suggestion
.. If the label is attached, remove the label from the project:
+
```

Not sure if "by entering/running the following command" is used throughout the module or not -- you might want to use it consistently in this file. 
Guideline https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command